qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -1037,6 +1103,10 @@ def save(self) -> None:
             for name, figure in self._figures.items():
                 if figure is None:
                     continue
+                # currently only the figure and its name are stored in the database
+                if isinstance(figure, FigureData):
+                    figure = figure.figure
+                    LOG.warning("Figure metadata is currently not saved to the database")
```

Comment:
This is the one that should have been changed to debug i think
```suggestion
                    LOG.debug("Figure metadata is currently not saved to the database")
```

Reply:
Fixed